pkg/network/node/networkpolicy.go

```diff
@@ -975,15 +1005,10 @@ func (np *networkPolicyPlugin) handleAddOrUpdatePod(obj, old interface{}, eventT
 	}
 
 	np.lock.Lock()
-	defer func() {
-		otx := np.node.oc.NewTransaction()
-		otx.DeleteFlows("table=27, cookie=1/-1, ip, nw_src=%s", pod.Status.PodIP)
-		otx.DeleteFlows("table=80, cookie=1/-1, ip, nw_src=%s", pod.Status.PodIP)
-		otx.Commit()
```

Comment:
So this needs to come back if we're keeping the `drop` flows... but not in `handleAddOrUpdatePod`; it needs to happen in `syncFlows`... (The existing code was wrong; it deletes the `drop` flows before adding the actual NetworkPolicy-related flows, so it creates a small gap where the pod is fully un-isolated...)

The use of `np.lock` ensures that if `syncFlows` sees `drop` rules for a particular IP, that it will also have been told to recalculate and resync all relevant namespace flows. And `SetUpPod` won't be able to create any new `drop` rules until after `syncFlows` returns. So that means that at the end of `syncFlows`, we can safely delete _all_ `drop` flows that had been created before that point, because we know that we now have the corresponding real NetworkPolicy flows for all of those pods. So it can just do:

		otx.DeleteFlows("table=27, cookie=1/-1")
		otx.DeleteFlows("table=80, cookie=1/-1")

well... we should probably use a more unique cookie value than "1"... Just pick a random number and define it as a const at the top and use it from both `SetUpPod` and `syncFlows`.